binance_trade_bot/binance_api_manager.py

```diff
@@ -136,15 +138,73 @@ def wait_for_order(self, order_id) -> BinanceOrder:
 
         self.logger.debug(f"Order created: {order_status}")
 
-        while order_status.status != "FILLED":
-            order_status = self.cache.orders.get(order_id, None)
-            self.logger.debug(f"Waiting for order {order_id} to be filled")
-            time.sleep(1)
+        while order_status["status"] != "FILLED":
```

Comment:
I don't think this is quite right, `order_status` should be fetched from the cache, and will be of type `BinanceOrder`, not a dictionary

Reply:
Thanks for pointing out. Debugging at the moment. Fixed this part. Trying to understand why pylint keeps skipping locally on my side...

Reply:
I updated the change. Ran normally too. Can you test?